toolkit/analysis_and_visualization/classification/assets/scripts/app.js

```diff
@@ -0,0 +1,578 @@
+            var showTo = showFrom + perPage;
+            loadImageResults(imageSummaryFiltered.slice(showFrom, showTo));
+
+        }
+    });
+
+
+
+
+    // We'll call this now so the initial items can be loaded.
+    updateItems();
+
+    // EDIT: Let's cover URL fragments (i.e. #page-3 in the URL).
+    // More thoroughly explained (including the regular expression) in: 
+    // https://github.com/bilalakil/bin/tree/master/simplepagination/page-fragment/index.html
+    // We'll create a function to check the URL fragment
+    // and trigger a change of page accordingly.
+    function checkFragment() {
+        // If there's no hash, treat it like page 1.
+        var hash = window.location.hash || "#page-1";
+        // We'll use a regular expression to check the hash string.
+        hash = hash.match(/^#page-(\d+)$/);
+        if (hash) {
+            // The `selectPage` function is described in the documentation.
+            // We've captured the page number in a regex group: `(\d+)`.
+            $(".pagination-page").pagination("selectPage", parseInt(hash[1], 10));
+        }
+    }
+
+    // We'll call this function whenever back/forward is pressed...
+    $(window).bind("popstate", checkFragment);
+    // ... and we'll also call it when the page has loaded
+    // (which is right now).
+    checkFragment();
+});
+
+
+
+
+
+function loadScorings() {
+    var myTableBody = $("#standard-scoring-table").find("tbody");
+    myTableBody.empty();
+    var row = $("<tr>");
+    data.scores.matchCounts.forEach(function (value) {
+        row.append($("<td>").text(value));
+    });
+    myTableBody.append(row);
+
+    row = $("<tr>");
+    data.scores.modelScores.forEach(function (value) {
+        row.append($("<td>").text(value.toFixed(2) + "%"));
+    });
+    myTableBody.append(row);
+
+
+    if (data.hasHierarchy) {
+        var tableIds = ["#method1-scoring-table", "#method2-scoring-table", "#method3-scoring-table"];
+        var scores = [data.scores.method1Scores, data.scores.method2Scores, data.scores.method3Scores];
+
+        tableIds.forEach(function (tableId, i) {
+            myTableBody = $(tableId).find("tbody");
+            myTableBody.empty();
+            var row = $("<tr>");
+
+            data.scores.matchCounts.forEach(function (value) {
+                row.append($("<td>").text(value));
+            });
+            myTableBody.append(row);
+
+            row = $("<tr>");
+            scores[i].forEach(function (value) {
+                row.append($("<td>").text(value.toFixed(2) + "%"));
+            });
+            myTableBody.append(row);
+
+
+        });
+        $("#hierarchial-scoring").toggleClass("hidden");
+    }
+
+
+}
+
+
+function compare(a, b, operator) {
+
+    switch (operator) {
+        case "contains":
+            return a.toString().toLowerCase().includes(b.toString().toLowerCase());
+        case "eq":
+            return a == b;
+        case "neq":
+            return a != b;
```

Comment:
Codacy found an issue: [Expected '!==' and instead saw '!='.](https://app.codacy.com/gh/GPUOpen-ProfessionalCompute-Libraries/MIVisionX/pullRequest?prid=6397592)